c++/src/ByteRLE.cc

```diff
@@ -203,6 +205,17 @@ namespace orc {
     recorder->add(static_cast<uint64_t>(numLiterals));
   }
 
+  void ByteRleEncoderImpl::suppress() {
+    // written data can be just ignored because they are only flushed in memory
+    outputStream->suppress();
+
+    numLiterals = 0;
```

Comment:
Why not nullify buffer? I'd add a reset() function to clear the state and use it here and in the ByteRleEncoderImpl constructor.

Reply:
I don't find the reset function, so I add this function by the way.